substrate/frame/contracts/src/migration.rs

```diff
@@ -161,9 +162,10 @@ impl<const N: u16> MigrationStep for NoopMigration<N> {
 	fn max_step_weight() -> Weight {
 		Weight::zero()
 	}
-	fn step(&mut self) -> (IsFinished, Weight) {
+	fn step(&mut self, meter: &mut WeightMeter) -> IsFinished {
 		log::debug!(target: LOG_TARGET, "Noop migration for version {}", N);
-		(IsFinished::Yes, Weight::zero())
+		meter.consume(Weight::zero());
```

Comment:
Is there a reason why we do that?

Reply:
not sure what I was thinking here, will update